packages/core/src/commands/up/index.ts

```diff
@@ -127,8 +126,6 @@ const up = async ({
     createCopiedFile('tunnel_server_public_key', formatPublicKey(hostKey)),
   ])
 
```

Comment:
Why remove this information? It can be useful

Reply:
it moved to line 97 together with the definition of `envId`